modules/appeals_api/spec/docs/higher_level_reviews_spec.rb

```diff
@@ -351,7 +351,7 @@
         produces 'application/json'
 
         response '200', 'the JSON Schema for POST /higher_level_reviews' do
-          it_behaves_like 'rswag example', desc: 'returns a 200 response'
+          it_behaves_like 'rswag example', desc: 'returns a 200 response', response_wrapper: :raw_body
```

Comment:
By getting the raw body as a string instead of a json/hash object, we avoid "invalid $ref" errors when validating the OAS with various tools like `swagger-cli` and `spectral`.